data/human/transport missions.txt

```diff
@@ -3582,3 +3582,297 @@ mission "Hauler VI cargo prototype 3"
+			choice
+				`	"I did my best."`
+					goto end
+				`	"No help is too much for a friend."`
+					goto end
+			label slow
+			`	The doctor that spoke to you earlier appears soon after you enter the building. "Alas! Such is our fate, to slowly pass from this life to the next!", he exclaims, and adds "Unfortunately, the disease was more onerous than we could have anticipated. The medicine you brought might save James's life, but the damage already wrought would leave him paralysed forever."`
+			apply
+				set "james medicine: james disabled"
+			choice
+				`	"I tried my best."`
+					goto end
+				`	"It's your fault, not ordering the medicine earlier."`
+			`	A heavy tear rolls down one side of the doctor's face. "How would I guess! He seemed quite well a few days ago, not showing any sign of the sickness spreading so fast!"`
+			label end
+			branch noname
+				not "james medicine: infected"
+			`	"Anyway, we have to pay for the services we receive. Here, take <payment>", he says and drops his voice to mention, "I would not recommend seeing James now, though I will surely tell him how helpful you have been, Captain <last>."`
+				goto finish
+			label noname
+			`	"Anyway, we have to pay for the services we receive. Here, take <payment>", he says and drops his voice to mention, "I would not recommend seeing James now, but maybe after he is well again."`
+			choice
+				`	"Sure! Can you him that captain <last> was looking for him?"`
+				`	"Oh yeah? I am going to visit him regardless."`
+					goto visit
+			`	"Of course, at the first possible occasion."`
+				goto finish
+			label visit
+			`	The doctor doesn't look very happy, but lets you go as there is nothing in the rules which would prevent you visiting James.`
+			apply
+				set "james medicine: infected"
+			`	You enter James's room but find him tight asleep. Not wanting to wake him up, you leave a note urging him to contact you when he is well again.`
+			`	Just before you leave the room, you notice a note written by him on one of the chairs. It reads "The North; Ultima Thule", leaving the reader to decide what its meaning is.`
+			label finish
+		payment 150000
+	
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+			confusion 20
+		system "Alpha Hydri"
+		fleet "Small Northern Pirates"
+
+mission "James Medicine [1] Preventive"
+	# Name is important; it should be lexically ordered after "James Medicine [1]"
+	invisible
+	landing
+	source "Nimbus"
+	to offer
+		has "James Medicine [0]: done"
+		has "james medicine: infected"
+	on offer
+		dialog
+			`Thinking about what the doctor said, it might be wise to buy some of the medicine for yourself. Checking the receipt you got, this would cost you 10,000 credits.`
+	on accept
+		payment -10000
+		set "james medicine: medicine taken"
+		fail
+	on decline
+		fail
```

Comment:
This is basically dead code. There's no mission to "fail" if it was "declined" in the first place. 
```suggestion
```